source/crud/delete.txt

```diff
@@ -51,6 +51,15 @@ To delete all documents from a collection, pass an empty
    more information and examples, see
    :method:`~db.collection.deleteMany()`.
 
+.. note::
+
+   If you want to delete all documents from a large collection, dropping
+   and recreating the collection may have faster performance than
+   deleting documents with the :method:`db.collection.deleteMany()`
+   method with an empty filter. When you recreate the collection, you
```

Comment:
[Optional]

Up to you, but I feel like we can remove "with an empty filter" here, since we talk about the empty filter behavior in the beginning of this section.

Reply:
Done!